roles/node/tasks/100-tests.yml

```diff
@@ -20,7 +20,10 @@
 - name: Test | Check node_chain
   ansible.builtin.fail:
     msg: "The 'node_chain' variable can't be empty!"
-  when: node_chain == '' and node_chainspec == ''
+  when:
+    - node_chain == ''
+    - node_chainspec == ''
+    - node_parachain_relay_chain_rpc_urls == ''
```

Comment:
This condition `node_parachain_relay_chain_rpc_urls == ''` is always `False`:
```
- hosts: localhost
  gather_facts: true
  vars:
    list_e: []
    list_f0: [""]
    list_f1: ["aaa"]
    list_f2: ["aaa","bbb"]
  tasks:
   - debug:
       msg: "condition list_e: {{ list_e == ''}}, list_f0: {{ list_f0 == '' }}, list_f1: {{ list_f1 == '' }}, list_f2: {{ list_f2 == '' }}"
```
output:
```
"msg": "condition list_e: False, list_f0: False, list_f1: False, list_f2: False"

```

Reply:
```suggestion
    - node_parachain_relay_chain_rpc_urls == []
```

Reply:
the condition should be:
(node_chain == '' and node_chainspec == '') or node_parachain_relay_chain_rpc_urls == []